src/mock-doc/element.ts

```diff
@@ -297,6 +314,25 @@ export class MockSVGElement extends MockElement {
   getTotalLength(): number {
     return 0;
   }
+  getComputedTextLength(): number {
+    return 0;
+  }
+  createSVGPoint() {
+    return {
+      matrixTransform: defaultSVGMatrix,
```

Comment:
Looking at https://www.w3.org/TR/SVG11/coords.html#InterfaceSVGPoint, should this be a callable function rather than an attribute on the object?

```suggestion
      matrixTransform: () => defaultSVGMatrix,
```